extension.js

```diff
@@ -64,40 +67,32 @@ function checkSettings() {
     }
 }
 
-let signals = [];
-let keybindings = [];
 function enable() {
-    settings = Convenience.getSettings();
-    signals.push(settings.connect('changed::' + Prefs.SETTINGS_REPLACE_OVERVIEW,
-                                  checkSettings));
+    // log('OverviewAllWindows enable');
+    settings = ExtensionUtils.getSettings();
+    signals.push(settings.connect('changed::' +
+            Prefs.SETTINGS_REPLACE_OVERVIEW, checkSettings));
     checkSettings();
 }
 
 function disable() {
+    // log('OverviewAllWindows disable');
     while (signals.length) {
         settings.disconnect(signals.pop());
     }
 
     while (keybindings.length) {
         _removeKeybindings(keybindings.pop());
     }
+    destroy();
```

Comment:
It seems like if we're going to call this here, then we should also be calling `setUp` in `enable`?

Reply:
Ooops... Yes, I missed that. Weird it seemed to work fine tho, but this probably explains other things that didn't work as I expected.